docs/source/code/spatialstats_model_vgm.py

```diff
@@ -0,0 +1,38 @@
+import matplotlib.pyplot as plt
```

Comment:
Do we actually need both examples? Thery're almost the same, so we should just keep the second one, no?

Reply:
Yes, work in progress!